src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,23 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.command.Command;
+import duke.task.Task;
+
+public class AddCommand extends Command {
+    
+    public Task task;
+
+    public AddCommand(Task task) {
+        super();
+        this.task = task;
+    }
+
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
```

Comment:
Although it is not required, it is recommended to add the '@ Override' annotation. [ https://stackoverflow.com/questions/1005898/should-i-add-an-override-annotation-when-implementing-abstract-methods-in-java](url)